xAPI.md

```diff
@@ -2798,6 +2815,8 @@ is included in [Appendix F: Table of All Endpoints](#AppendixF).
 * The LRS MUST support all of the endpoints described in [this section](#datatransfer). 
 * If the LRS implements OAuth 1.0, the LRS MUST also support all of the OAuth endpoints 
 described in [Section 6.4.2 OAuth Authorization Scope](#oauthscope).
+* Other systems, which are not LRS, MAY implement the LRS requirements for one or more of 
```

Comment:
I think it's helpful to note it, but that's done in the descriptive text above, so I'll remove this requiredment.